test/unit/processing/due-dates/confirm-due-dates.test.js

```diff
@@ -0,0 +1,129 @@
+const { createAdjustmentPaymentRequest } = require('../../../helpers')
```

Comment:
Why is this one defined at the top of the file?

Reply:
It's a helper function for the tests.  To be honest wasn't sure what a natural position for these were but opted to put them at them above the mocks to show that we're also dependent on some extra stuff to help with the tests here.  A bit like when dependent on third party stuff, but less dramatic